scalyr_agent/configuration.py

```diff
@@ -824,6 +825,11 @@ def debug_level(self):
         """Returns the configuration value for 'debug_level'."""
         return self.__get_config().get_int("debug_level")
 
+    @property
+    def stdout_severity(self):
+        """Returns the configuration value for 'stdout_severity'."""
```

Comment:
I would add a note here that this is only used if the agent process is not forced.